src/StudioCore/Assets/Paramdex/ER/Defs/AssetGeometryParam.xml

```diff
@@ -198,12 +198,21 @@
       <Maximum>1</Maximum>
       <SortID>2501</SortID>
     </Field>
-    <Field Def="dummy8 Reserve_2:1">
+    
+    <Field Def="dummy8 Reserve_2:1" RemovedVersion="11210015">
```

Comment:
I was about to say "in the case of dummies, surely we can just rename the field?" but considering we did have an original paramdef, it may be best to preserve it as an effective guarantee on fromsoft-level that is was dummy at that time.